### PR TITLE
Add getLoanScheme rpc

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Add getLoanScheme rpc

#### Which issue(s) does this PR fixes?:

Fixes part of #488

### CODE REVIEW

File: packages/jellyfish-api-core/src/category/loan.ts [hunk 1]
```diff
@@ -62,12 +62,22 @@ export class Loan {
...
    */
-  async listLoanSchemes (): Promise<LoanSchemeResult[]> {
+  async listLoanSchemes (): Promise<ListLoanSchemeResult[]> {
     return await this.client.call('listloanschemes', [], 'bignumber')
   }
 
+  /**
+   * Get loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme, max 8 chars
```
Comment (Jouzo) [inline packages/jellyfish-api-core/src/category/loan.ts:74]: the max 8 chars is enforced on creation, right ? 
It is confusing here.

Comment (jingyi2811) [reply packages/jellyfish-api-core/src/category/loan.ts:74]: Don't think it is confusing. This is the same description for id across loan rpcs

Comment (Jouzo) [reply packages/jellyfish-api-core/src/category/loan.ts:74]: If you request an id with more than 8 chars, it won't raise any error. It just won't find any loan scheme.

Comment (jingyi2811) [reply packages/jellyfish-api-core/src/category/loan.ts:74]: Hmm, I shall inform C++ programmer to fix this.

File: packages/jellyfish-api-core/__tests__/category/loan/loan_container.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontain
 
 export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
   constructor () {
-    super(undefined, 'defi/defichain:HEAD-faf9e78')
+    super(undefined, 'defi/defichain:HEAD-a7e7ac1')
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/__tests__/category/loan/loan_container.ts:5]: I update to the latest docker image

File: packages/jellyfish-api-core/src/category/loan.ts [hunk 3]
```diff
@@ -68,6 +68,16 @@ export class Loan {
     return await this.client.call('listloanschemes', [], 'bignumber')
   }
 
+  /**
+   * Get loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/src/category/loan.ts:74]: I just confirmed with Diego. He said that there is no checking on whether the id is an empty string or more than characters in C++ side.

File: packages/jellyfish-api-core/src/category/loan.ts [hunk 4]
```diff
@@ -68,6 +68,16 @@ export class Loan {
     return await this.client.call('listloanschemes', [], 'bignumber')
   }
 
+  /**
+   * Get loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme, max 8 chars.
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/src/category/loan.ts:74]: @Jouzo It is max 8 chars now.

**Review by Jouzo — APPROVED** [review]

**Review by surangap — APPROVED** [review]

**Comment by surangap** [comment]
@jingyi2811 
please check why some of the earlier tests failed.

**Comment by jingyi2811** [comment]
@surangap All good now :)

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**